Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -399,10 +406,8 @@ CFTypeID CGContextGetTypeID() {
 #pragma region Global State - Lifetime
 static HRESULT __CGContextPrepareDefaults(CGContextRef context) {
     // Reference platform defaults:
-    // * Fill  : fully transparent black
-    // * Stroke: fully opaque black
-    // If a context does not support alpha, the default fill looks like fully opaque black.
-    CGContextSetRGBFillColor(context, 0, 0, 0, 0);
+    // * All colors are fully opaque black.
```

Comment:
The commit it is in is titled "Fix the default ... from the reference platform."